.github/workflows/linux.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           python3.7-dbg -m pip install --upgrade pip setuptools wheel cython pytest pytest-xdist argparse pybind11
           python3.7-dbg -m pip install --upgrade mpmath gmpy2
           python3.7-dbg -m pip install --upgrade --no-cache-dir numpy
+          python3.7-dbg -m pip install --upgrade pythran
```

Comment:
Let's put this on the same line as `cython` and other build-related packages, will be faster